docs/source/development/linting.md

```diff
@@ -0,0 +1,99 @@
+# Linting
+
+## Introduction
+Linting tools are used to improve the quality of your code by checking for errors and making sure it meets a stylistic
+standard before it is run. As a project grows and goes through various stages of development, it becomes important to
+maintain the code quality. Linting makes your code more readable, easy to debug and maintain, and stylistically
+consistent.
+
+## Set up linting tools
+There are a variety of linting tools available to use with your Kedro projects. This guide shows you how to use
+[`black`](https://github.com/psf/black), [`flake8`](https://gitlab.com/pycqa/flake8), and
+[`isort`](https://github.com/PyCQA/isort) to lint your Kedro projects.
+- **`black`** is a [PEP 8](https://peps.python.org/pep-0008/) compliant opinionated Python code formatter. `black` can
+check for styling inconsistencies and reformat your files in place.
+[You can read more in the `black` documentation](https://black.readthedocs.io/en/stable/).
+- **`flake8`** is a wrapper around [`pep8`](https://pypi.org/project/pep8/),
+[`pyflakes`](https://pypi.org/project/pyflakes/), and [`mccabe`](https://pypi.org/project/mccabe/) which can flag
+programming errors and coding style inconsistencies with respect to [PEP 8](https://peps.python.org/pep-0008/),
+and check the cyclomatic complexity of your code base.
+[You can read more in the `flake8` documentation](https://flake8.pycqa.org/en/latest/).
+- **`isort`** is a Python library used to sort imports alphabetically, and automatically separated into sections and by
+type. [You can read more in the `isort` documentation](https://pycqa.github.io/isort/).
+
+### Install linting tools
+You can install `black`, `flake8`, and `isort` by adding the following lines to your project's `src/requirements.txt`
+file:
+```text
+black # Used for formatting code
+flake8 # Used for linting code
+isort # Used for linting code
+```
+To install all the project-specific dependencies, including the linting tools, navigate to the root directory of the
+project and run:
+```bash
+pip install -r src/requirements.txt
+```
+Alternatively, you can individually install the linting tools using the following shell commands:
+```bash
+pip install black
+pip install flake8
+pip install isort
+```
+
+### Run linting tools
+Use the following commands to run lint checks:
+```bash
+black --check <project_root>
+flake8 <project_root>
+isort --profile black --check <project_root>
+```
+You can also have `black` and `isort` automatically format your code by omitting the `--check` flag. Since `isort` and
+`black` both format your imports, adding `--profile black` to the `isort` run helps avoid potential conflicts.
+
+## Automating linting with `pre-commit` hooks
+
+You can also automate linting by using [`pre-commit`](https://github.com/pre-commit/pre-commit) hooks.
```

Comment:
```suggestion
[`pre-commit`](https://github.com/pre-commit/pre-commit) hooks can be used to automate linting.
```

Sorry, this might be a bit pedantic I think we should avoid starting with 'You can also'

Reply:
Thanks @SajidAlamQB for the review! 
Do you think I could remove the 'also' instead - "You can automate linting by using `pre-commit` hooks"? 
The [contribute to kedro docs](https://kedro.readthedocs.io/en/stable/contribution/documentation_contributor_guidelines.html#style) guidelines mention avoiding passive tense.

Reply:
Yes, that seems good as well.

Reply:
Updated in 969c6dc7102657e8a145a83c2de02ae6f8004478!